cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -472,6 +473,14 @@ storage:
               - --validate-application-label-min-creation-time={{ .Cluster.ConfigItems.teapot_admission_controller_application_min_creation_time }}
               - --application-registry-url=http://127.0.0.1:8285/
 {{- end }}
+              - --pod-inject-environment-variables
+              - --pod-environment-variable=_PLATFORM_ACCOUNT={{ .Cluster.Alias }}
+              - --pod-environment-variable=_PLATFORM_TOKENINFO_URL={{ .ConfigItems.tokeninfo_url }}
+
+              - --pod-environment-variable=_PLATFORM_OPENTRACING_TAG_ACCOUNT={{ .Cluster.Alias }}
+              - --pod-environment-variable=_PLATFORM_OPENTRACING_LIGHTSTEP_COLLECTOR_PORT=8443
+              - --pod-environment-variable=_PLATFORM_OPENTRACING_LIGHTSTEP_COLLECTOR_HOST=tracing.stups.zalan.do
+              - --pod-environment-variable=_PLATFORM_OPENTRACING_LIGHTSTEP_ACCESS_TOKEN={{ .ConfigItems.lightstep_token }}
```

Comment:
This will expose the token to everyone with read access in the cluster. We should think of a way to configure this such that normal users can't just read the token. Unfortunately we can't use secrets afaik because this is a mirror pod.

Any ideas how to solve this?

Reply:
as discussed elsewhere:
- token can only be used for writing
- token is not security relevant